core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/rules/DeleteOnMultipleColumns.scala

```diff
@@ -0,0 +1,43 @@
+package com.databricks.labs.remorph.parsers.snowflake.rules
+
+import com.databricks.labs.remorph.intermediate._
+
+class DeleteOnMultipleColumns extends Rule[LogicalPlan] {
```

Comment:
[Tech Debt]: For now this is fine, But I think we move this as an optimizer for filter with multiple columns.